app/controllers/subject_viewer.coffee

```diff
@@ -150,7 +150,14 @@ class SubjectViewer extends Controller
   #   delete @mouseDown
 
   onClickPlay: ->
-    AnalyticsLogger.logEvent 'play', @classification.id, @classification.subject.zooniverseId
+    Geordi.logEvent {
+      type: 'play'
+      relatedID: @classification.id
```

Comment:
Relevant throughout this file: `@classification.id` isn't defined until the classification has been saved.